stockservice-go/internal/core/ports/ports.go

```diff
@@ -0,0 +1,47 @@
+package ports
+
+import (
+	"context"
+	"github.com/ZiyadBouazara/bitcoin-pulse/stockservice-go/internal/core/domain"
+	"net"
+)
+
```

Comment:
i believe since this is an open source project having comments out here in the interface configuration part of the app could help new collaborators have a better global understanding of the app and what each service & it's components do.